bpf/bpf_host.c

```diff
@@ -879,6 +880,10 @@ int from_netdev(struct __ctx_buff *ctx)
 __section("from-host")
 int from_host(struct __ctx_buff *ctx)
 {
+	/* Traffic from the host ns going through cilium_host device must
+	 * not be subject to EDT rate-limiting.
+	 */
+	edt_set_aggregate(ctx, 0);
```

Comment:
Not necessarily, for example, consider the case when we have tunneling enabled, this would go via cilium_host -> cilium_vxlan -> phys dev. In this case we want to make sure that EDT aggregate is 0 and not subject to rate-limiting, too.